src/implementation/Server/GRPCServer/pubsub.ts

```diff
@@ -26,8 +27,8 @@ export default class DaprPubSub implements IServerPubSub {
     this.logger = new Logger("GRPCServer", "PubSub", server.client.options.logger);
   }
 
-  async subscribe(pubSubName: string, topic: string, cb: TypeDaprPubSubCallback): Promise<void> {
-    this.logger.info(`Registering onTopicEvent Handler: PubSub = ${pubSubName}; Topic = ${topic}`)
-    this.server.getServerImpl().registerPubSubSubscriptionHandler(pubSubName, topic, cb);
+  async subscribe(pubSubName: string, topic: string, cb: TypeDaprPubSubCallback, _route?: "", metadata?: KeyValueType): Promise<void> {
+    this.logger.info(`Registering onTopicEvent Handler: PubSub = ${pubSubName}; Topic = ${topic}; Metadata: ${metadata}`)
+    this.server.getServerImpl().registerPubSubSubscriptionHandler(pubSubName, topic, cb, metadata ?? {});
```

Comment:
Shouldn't we just pass metadata and make it default in the `registerPubSubSubscriptionHandler`?

Reply:
Same as above, let me know if you want me to change it.